demo/index.js

```diff
@@ -120,15 +130,7 @@ const Demo = enhance(
               ? ({ rotation, scale, opacity, top: topOffset }) => (
                   <Popper
                     placement="bottom"
-                    modifiers={{
```

Comment:
the warning allowed me to catch us over updating the modifiers object in the demo here!